### PR TITLE
Add child page from userbar, as discussed in #694

### PR DESCRIPTION
**Pull request by edrex**
Add child page from userbar, as discussed in #694

### CODE REVIEW

**Comment by chrxr** [comment]
Hi @edrex, thanks for the changes. @kaedroho to review.

File: wagtail/wagtailadmin/userbar.py [hunk 1]
```diff
@@ -24,7 +24,7 @@ def render(self, request):
             return ""
 
         # Don't render if user doesn't have ability to add siblings
```
Comment (kaedroho) [inline wagtail/wagtailadmin/userbar.py]: Comment should be changed here "# Don't render if user doesn't have ability to add **children**"

**Comment by kaedroho** [comment]
Thanks @edrex these changes look good to me and work well! :+1:

**Comment by kaedroho** [comment]
Merged in https://github.com/torchbox/wagtail/commit/c87802b68b435a8148d04e9fd0182b37c59cfd97

**Pull request closed without merging**